types/bn.js/index.d.ts

```diff
@@ -62,7 +66,17 @@ declare class BN {
     /**
      * @description returns true if the supplied object is a BN.js instance
      */
-    static isBN(b: any): b is BN;
```

Comment:
Why was this changed?

Reply:
I had a misunderstanding about the `b is BN` type. I've restored it.